$NetBSD: patch-ah,v 1.3 2009/06/07 11:23:47 wiz Exp $

This patch provides a black background by default--a little
nicer, no?

--- dix/window.c.orig	2009-07-26 16:56:57.000000000 -0400
+++ dix/window.c
@@ -138,8 +138,12 @@ Equipment Corporation.
  *    ChangeWindowDeviceCursor
  ******/
 
-static unsigned char _back_lsb[4] = {0x88, 0x22, 0x44, 0x11};
-static unsigned char _back_msb[4] = {0x11, 0x44, 0x22, 0x88};
+/* TODO: Perhaps we should allow the user/system administrator
+ *	to set this via environment variable or configuration
+ *	directive?
+ */
+static unsigned char _back_lsb[4] = {0x00, 0x00, 0x00, 0x00};
+static unsigned char _back_msb[4] = {0x00, 0x00, 0x00, 0x00};
 
 static Bool WindowParentHasDeviceCursor(WindowPtr pWin, 
                                         DeviceIntPtr pDev, 
